app/services/acuant/acuant_client.rb

```diff
@@ -29,26 +43,22 @@ def post_images(front_image:, back_image:, instance_id: nil)
       back_response = post_back_image(image: back_image, instance_id: instance_id)
       response = merge_post_responses(front_response, back_response)
 
-      check_results(response, instance_id)
+      results = check_results(response, instance_id)
+
+      if results.success?
+        pii = results.pii_from_doc
+        selfie_response = post_selfie(image: selfie_image, instance_id: instance_id)
+        [selfie_response, pii]
+      else
+        results
```

Comment:
Looks like this line is uncovered. I think it makes sense for us to drop in a unit test for what happens if posting the selfie fails.